transferwise/pipelinewise-singer-python
# Logging

# Description of change
Get rid of logging config to allow any libraries that use this package to define the wanted logging config.
Name the local logger as `singer` to be able to identify it.

Added a way to provide a configuration file if desired, it's via providing the path to the `logging.conf` file with the help of the environment variable `LOGGING_CONF_FILE`

# Manual QA steps
 - 1st test: run a tap using these changes and observe the logs: there would be no logs due to not having any config
 - 2nd test :set the `LOGGING_CONF_FILE` to point to a logging config file and run a tap with these changes: logs would show up.
 
# Risks
 - None
 
# Rollback steps
 - Revert this branch
